src/gpuocean/SWEsimulators/CDKLM16.py

```diff
@@ -603,6 +607,24 @@ def step(self, t_end=0.0, apply_stochastic_term=True, write_now=True, update_dt=
 
     def drifterStep(self, dt):
         # Evolve drifters
+        if self.hasCPdrifters:
```

Comment:
Two things:
1) Rename `hasCPdrifters `to `hasCrossProdDrifters`? 
2) Can the content of this if block be put into a separate function? For example `self.driftCrossProdDrifters(...)`

Reply:
Renamed. I think the separate function is not necessary